app/api/v2/__init__.py

```diff
@@ -0,0 +1,15 @@
+from flask import Blueprint
+from flask_restplus import Api
+from.views.orders import v2_order
+from .views.user import v2_user
+
+v2_blueprint = Blueprint('v2_blueprint', __name__,url_prefix='/api/v2')
+v2_api = Api(v2_blueprint,
+             title="SendIT",
```

Comment:
Kindly stick to either single quotes or double quotes, but don't use both in your project.